montblanc/tests/test_meq_tf.py

```diff
@@ -239,32 +239,28 @@ def point_lm(self, context):
         return pt_lm[lp:up, :]
 
     def point_stokes(self, context):
-        (lp, up), (lt, ut) = context.dim_extents('npsrc', 'ntime')
-        return np.tile(pt_stokes[lp:up, np.newaxis, :], [1, ut-lt, 1])
+        (lp, up), (lt, ut), (lc, uc) = context.dim_extents('npsrc', 'ntime', 'nchan')
+        # (npsrc, ntime, nchan, 4)
+        s = pt_stokes[lp:up,None,None,:]
+        a = np.broadcast_to(pt_alpha[lp:up,None,None,None], (up-lp,ut-lt,1,1))
+        rf = pt_ref_freq[lp:up,None,None,None]
+        f = frequency[None,None,lc:uc,None]
 
-    def point_alpha(self, context):
-        (lp, up), (lt, ut) = context.dim_extents('npsrc', 'ntime')
-        return np.tile(pt_alpha[lp:up, np.newaxis], [1, ut-lt])
-
-    def point_ref_freq(self, context):
-        (lp, up) = context.dim_extents('npsrc')
-        return pt_ref_freq[lp:up]
+        return s*(f/rf)**a
```

Comment:
@bennahugo See above for how to input stokes parameters along with the default spectral index into montblanc